Framework/API/src/MultiDomainFunction.cpp

```diff
@@ -190,32 +190,6 @@ void MultiDomainFunction::iterationFinished() {
   }
 }
 
-/// Return a value of attribute attName
-IFunction::Attribute
-MultiDomainFunction::getLocalAttribute(size_t i,
-                                       const std::string &attName) const {
-  if (attName != "domains") {
-    throw std::invalid_argument("MultiDomainFunction does not have attribute " +
-                                attName);
-  }
-  if (i >= nFunctions()) {
-    throw std::out_of_range("Function index is out of range.");
-  }
-  auto it = m_domains.find(i);
-  if (it == m_domains.end()) {
-    return IFunction::Attribute("All");
-  } else if (it->second.size() == 1 && it->second.front() == i) {
-    return IFunction::Attribute("i");
-  } else if (!it->second.empty()) {
-    std::string out(boost::lexical_cast<std::string>(it->second.front()));
-    for (auto i = it->second.begin() + 1; i != it->second.end(); ++it) {
```

Comment:
I just wanted to point out that he current implementation is most likely broken anyway. I think it is better to just remove the unused member function.